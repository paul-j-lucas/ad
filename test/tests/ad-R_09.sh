# /bin/sh

ME=`basename $0`

[ `uname` == Darwin ] ||
  { echo "$ME: this test is supported only on macOS" >&2; exit 1; }

INPUT=/tmp/rsrc_fork_$$_
trap "x=$?; rm -f /tmp/*_$$_*; exit $x" EXIT HUP INT TERM
OUTPUT=$1
LOG_FILE=$2

touch $INPUT
xattr -w com.apple.ResourceFork 'hello, world' $INPUT

ad -R $INPUT $OUTPUT
diff expected/ad-R_09.txt $OUTPUT > $LOG_FILE

# vim:set et sw=2 ts=2:
