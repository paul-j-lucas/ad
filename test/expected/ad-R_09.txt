0000000000000000: 6865 6C6C 6F2C 2077  6F72 6C64            hello, world
